resource_retriever/test/test.cpp

```diff
@@ -31,16 +31,26 @@
 
```

Comment:
```suggestion
#include <memory>
#include <vector>
```

(and alphabetize it with the above)